libpod/container_internal_linux.go

```diff
@@ -98,6 +98,26 @@ func (c *Container) generateSpec(ctx context.Context) (*spec.Spec, error) {
 		}
 	}
 
+	for _, m := range g.Mounts() {
```

Comment:
Throw a comment on here about what's going on to make it obvious

Reply:
Added